library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -140,6 +148,14 @@ public class ChuckerInterceptor private constructor(
             this.decoders += decoder
         }
 
+        /**
+         * If set to `true`, [ChuckerInterceptor] will create a shortcut for your app
+         * just like what `LeakCanary` did, you can easily to access chucker's [MainActivity]
```

Comment:
I'm not sure if we should mention `LeakCanary` here. Don't get me wrong, it is a great tool, but there's nothing special about `LeakCanary` when it comes to this feature. Maybe link to Android docs instead and highlight that it is a dynamic shortcut?

Reply:
Removed that